### ISSUE TITLE
Pod Install not working

### ISSUE THREAD

**Issue by MHX792** [spokestack/spokestack-ios#88]
Error:

```
[!] Error installing Spokestack-iOS
[!] /usr/bin/git clone https://github.com/spokestack/spokestack-ios.git /var/folders/71/9tdn80h96cv0fgznfwlmsc0w0000gn/T/d20200815-21149-5hz3pc --template= --single-branch --depth 1 --branch 12.0.1

Cloning into '/var/folders/71/9tdn80h96cv0fgznfwlmsc0w0000gn/T/d20200815-21149-5hz3pc'...
Note: checking out '48c688301f8ee6966a011fa5cd267acc18a985e9'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

git-lfs filter-process: git-lfs: command not found
fatal: the remote end hung up unexpectedly
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'
```

**Comment by noelweichbrodt** [comment]
Thanks for the report! It appears that this release included some inadvertent Git LFS artifacts. 

To workaround temporarily while we investigate, please ensure that Git LFS is installed and then try the `pod install` again: https://git-lfs.github.com/.

**Comment by noelweichbrodt** [comment]
This will be fixed in the next release, 13.2!
